webdriver-spec.html

```diff
@@ -3138,96 +3148,75 @@
           with the <a>context object</a> equal to the <var>start node</var>.
       </dl>
     <li><p>Return <var>elements</var>.
-
   </ol>
-</section> <!-- /CSS Selector -->
 
-<p class=issue><strong>Warning:</strong>
- This rest of this section has not yet been redefined to match the <a>routing requests</a> model,
- and uses old concepts and definitions.
- Please do not rely on it yet.
+<h3>Link Text</h3>
+<p>To find a <a>WebElement</a> with the <dfn data-lt="link text selector">Link Text</dfn>
+   <a>strategy</a> the following steps need to be completed:
 
-<section>
-<h3>ECMAScript</h3>
+<ol>
+  <li><p>let <var>elements</var> be the result of calling <a>querySelectorAll</a>,
+    with argument <a>a elements</a>, with the <a>context object</a> equal to the
+    <var>start node</var>.
 
-<p>Finding elements by ecmascript is covered in the <a href="#ecmascript">ecmascript part of this spec</a>.
-</section>
+  <li><p>Let <var>result</var> be an empty <a>NodeList</a>.
 
-<section>
-<h3>Link Text</h3>
+  <li><p>For each <var>element</var> in <var>elements</var>:
+    <ol>
+      <li><p>Let <var>rendered text</var> be
+       the result of <a>getting <code>innerText</code></a>
+       of <var>element</var>.
 
-<p>Strategy name: link text</p>
+      <li><p>If <var>rendered text</var> equals <var>selector</var>, append
+        <var>element</var> to <var>result</var>.
+    </ol>
 
-<p>This strategy MUST be supported by all WebDriver implementations. </p>
+  <li><p>Return <var>result</var>.
+</ol>
 
-<p>This strategy MUST return all <a href="http://www.w3.org/TR/html5/text-level-semantics.html#the-a-element">"a"</a> elements with visible text exactly and case sensitively equal to the term being searched for.</p>
+<h3>Partial Link Text</h3>
 
-<div class="note">
-      <p>This is equivalent to:</p>
-      <pre>
-            elements = driver.find_elements(by = By.TAG_NAME, value = "a")
-            result = []
-            for element in elements:
-              text = element.text
-              if text == search_term:
-                result.append(element)
-          </pre>
-          <p>Where "search_term" is the link text being searched for, and "result" contains the list of elements to return.</p>
-    </div>
-</section> <!-- /Link Text -->
+<p>The <dfn data-lt="partial link text selector">Partial Link text</dfn> <a>strategy</a>
+  is very similar to the <a>Link Text</a> <a>strategy</a>, but rather than matching
+  the entire string, only a subset needs to match. That is, this MUST return all
+  <a>a elements</a> with rendered text that partially or completely and case
```

Comment:
I think we should codify this to `<a><code>a</code> elements</a>` for clarity throughout.